File: sop-sdk/sdk-java/src/main/java/com/gitee/sop/sdk/request/CommonRequest.java
package com.gitee.sop.sdk.request;

import com.gitee.sop.sdk.response.CommonResponse;

/**
 * @author tanghc
 */
public class CommonRequest extends BaseRequest<CommonResponse> {

    public CommonRequest(String method) {
        super(method, null);
    }

    public CommonRequest(String method, String version) {
        super(method, version);
    }

    @Override
    protected String method() {
        return "";
    }
}

File: sop-sdk/sdk-csharp/SDKCSharp/Request/CommonRequest.cs
using System;
using SDKCSharp.Response;

namespace SDKCSharp.Request
{
    public class CommonRequest : BaseRequest<CommonResponse>
    {
        public CommonRequest(string method): base(method, null)
        {
           
        }

        public CommonRequest(string method, string version): base(method, version)
        {
           
        }

        public override string GetMethod()
        {
            return "";
        }

    }
}
